templates/microprofile-openliberty/files/.platform.app.yaml

```diff
@@ -39,4 +39,4 @@ web:
     commands:
         start: |
            cp target/microprofile-liberty.jar server/microprofile-liberty.jar
-           java -jar -Xmx512m -Ddefault.http.port=8888 server/microprofile-liberty.jar
\ No newline at end of file
+           java -jar -Xmx512m -Ddefault.http.port=$PORT server/microprofile-liberty.jar
```

Comment:
This change can/should be in its own PR.  No need to block it on all of these others.

Reply:
Agreed